zbus/src/connection/socket_reader.rs

```diff
@@ -63,6 +68,9 @@ impl SocketReader {
                         match rule.matches(msg) {
                             Ok(true) => (),
                             Ok(false) => continue,
+                            #[cfg(not(feature = "tracing"))]
+                            Err(_e) => continue,
+                            #[cfg(feature = "tracing")]
```

Comment:
This turned out to be more intrusive than I was hoping for. Why do we need feature gating in the rest of the code, if you created the macros for both cases (feature enabled and disabled)? :thinking: